### DISCUSSION TITLE
TShock 1.4.2.3

### DISCUSSION THREAD

**Discussion by Mewsen** [Pryaxis/TShock#2313] [Q&A] [answered]
Terraria just got updated to 1.4.2.3

**Comment by Kobatz** [comment]
Eventually will be updated. I downloaded 1.4.2.2 off the Steam Depot so I could still play.

**Comment by ghost** [comment]
we can just wait

**Comment by moisterrific** [comment]
The Open Terraria API that TShockAPI runs on is already compatible with the latest client. However, I'm not sure what new features ReLogic has added that TShock needs to be updated to account for.

**Reply by HeheXD-bit** [reply]
When I log on to the server, it says you are not using the same version as this server. (1.4.2.2) How do I fix that?

**Reply by bartico6** [reply]
Because there is no 1.4.2.3 release out right now. @moisterrific has only stated that there is code in place to enable 1.4.2.3 to work.

**Comment by hakusaro** [comment, marked as answer]
It's updated! See: https://github.com/Pryaxis/TShock/releases/tag/v4.5.3.
